packages/material-ui/src/styles/colorManipulator.d.ts

```diff
@@ -11,7 +11,7 @@ export function decomposeColor(color: string): ColorObject;
 export function recomposeColor(color: ColorObject): string;
 export function getContrastRatio(foreground: string, background: string): number;
 export function getLuminance(color: string): number;
-export function emphasize(color: string, coefficient?: number): string;
+export function emphasize(color: string, coefficient: number): string;
```

Comment:
Should we remove the default argument of emphasize if we make it required?

https://github.com/mui-org/material-ui/blob/34337e89ade81ef5d72da878eeb1700a3839e99e/packages/material-ui/src/styles/colorManipulator.js#L197